src/Main.cpp

```diff
@@ -96,6 +96,18 @@ int main(int argc, char** argv)
         }
         std::cout << wp_device.getConfig() << std::endl;
     }
+    else if (cmd == "set_id") {
```

Comment:
Also update the usage message at the top of the file please (add the `dc-cube` command as well)